APE24.rst

```diff
@@ -0,0 +1,507 @@
+`OpenAstronomy packaging guide`_ for a few years,
+and `Astropy package template <https://github.com/astropy/package-template>`_ before that.
+
+pyOpenSci is also developing a `community-driven packaging guide`_
+that covers modern best practices and recommendations for scientific Python packaging. 
+This guide has a stringent community review process. Reviewers for this guide
+represent members of PyPA, core Python, Anaconda (conda/conda-forge), and
+core packages for front- and back-end tools (e.g., ``flit``, PDM, ``hatch``/``hatchling``, etc.).  
+
+If we partner with pyOpenSci, we encourage our Affiliated packages to continue to follow the
+`OpenAstronomy packaging guide`_ for packaging structure, which ensures a packaging
+style consistent with the majority of existing Affiliated packages as well as much of
+the core scientific Python environment Astropy relies on, in particular Numpy and Scipy.
+However maintainers will find the pyOpenSci guide to be a useful reference, given it provides
+community-wide guidelines and is targeted to those newer to packaging (e.g., just creating the initial package for their code).
+Editors and reviewer shall not let a particular choice of packaging influence their rating of
+a package.
+
+Regardless, the
+`Editor-in-Chief checks <https://www.pyopensci.org/software-peer-review/how-to/editor-in-chief-guide.html#editor-checklist-template>`_
+are the bare minimum for a package being able to go through the pyOpenSci review process.
+
+.. _OpenAstronomy packaging guide: https://packaging-guide.openastronomy.org/en/latest/
+
+.. _community-driven packaging guide: https://www.pyopensci.org/python-package-guide/
+
+Listing on websites
+-------------------
+
+The pre-APE 24 `Astropy Affiliated Packages`_ listing are listed by alphabetical order
+with hard-coded badges. While these badges are color-coded to give a at-a-glance status
+of packages, they become outdated over time and now could even be misleading
+(i.e., they really are showing the status at the time of acceptance, not the current status).
+pyOpenSci is willing to work with Astropy and the broader scientific community to develop
+a more consistent standard of evaluating the "health state" of a package, using
+universally accepted metrics for such measurements (see **"Re-review" of packages**);
+therefore, getting rid of the need for static badges altogether.
+
+pyOpenSci is willing to create the following specifically for Astropy
+if we agree to this partnership:
+
+* A feed of Affiliated packages that we could ingest and
+  use to rebuild our own `Astropy Affiliated Packages`_ listing.
+  This way, the packages would be listed both over at pyOpenSci and at Astropy
+  websites.
+* An Astropy-specific page on the pyOpenSci website that would be a dedicated
+  link for only Affiliated packages. We could then include some other
+  information about the Astropy ecosystem and branding (logo, colors, etc).
```

Comment:
Looking back at https://github.com/astropy/astropy-project/issues/334#issuecomment-1551657063 it is perhaps not clear yet how this representation would look for the currently existing (not pyOS-approved) Affiliated packages – would they just remain in our listing, maybe separate from the new feed (which may be technically inevitable anyway), or would there be also another link back from the pyOS page?
In principle Affiliated packages could be in one of 3-4 states in that phase:
1. Accepted under the new system (first-time or re-reviewed)
2. Previously accepted and not re-reviewed
3. Re-reviewed and sunsetted
4. [re-reviewed and found still conforming to old Astropy standards, but not to pyOpenSci ones]

Reply:
> it is perhaps not clear yet how this representation would look for the currently existing (not pyOS-approved) Affiliated packages

All the existing packages would be in a static "legacy" page (basically the current Affiliated page but we would stop updating it going forward and slap a giant "legacy" note on top).

I suspect they would have to "re-apply" via the new process to be listed in the pyOpenSci/Astropy page, especially if they were accepted more than 2 years ago. 

I tried to explain this in the `Implementation` section on Step 3. But as you can see, I am also not very sure about it.

Do you think we need a new section just for this? Do you have a better idea on how to do this?